README.md

```diff
@@ -1,10 +1,62 @@
 # Macarena Finance
 ![](./public/og.png)
 
-Macarena finance is simply a UI for Yearn Finance that you actually want to fork.
+Macarena finance is a simple UI for Yearn Finance, made to be forked!
 
-## How to run the project  
-1. Clone the repository  
-2. Run `yarn`  
-3. Run `yarn run dev`  
-4. Access `http://localhost:3000`  
+Running your own instance of Yearn makes you eligible to earn fees on top of all deposits made through your UI. See information on how partnership and profit-sharing work at our [Partner Docs](https://docs.yearn.finance/partners/introduction#profit-share-model)
+
+## Quickstart
+
+**Live Demo:** https://macarena.finance/
+
+1. Clone the repository: `git clone https://github.com/yearn/macarena-finance.git`
+2. Run `yarn` to install dependencies
+3. Run `yarn run dev` to raise the developer environment
+4. Open your browser at `http://localhost:3000`  
+
+**Configuring live environment to receive profit fees**
+
+5. At [`next.config.js`](./next.config.js) change `PARTNER_ID_ADDRESS` to the address that should receive the partner fees
```

Comment:
you can also set an environment variable and no need to modify this file. Would you mind adding it as first option?

Reply:
I talked with major and he said that this variable is not available to override through env vars, but I saw that there are some env vars you can add so I'll document them here as well!